scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOPredictionWordEmbedding.scala

```diff
@@ -30,13 +31,16 @@ trait H2OMOJOPredictionWordEmbedding {
     val schema = getWordEmbeddingPredictionSchema()
     val function = (r: Row) => {
       val model = H2OMOJOCache.getMojoBackend(uid, getMojo, this)
-      val pred = model.predictWord2Vec(RowConverter.toH2ORowData(r))
-      new GenericRowWithSchema(Array(pred.wordEmbeddings), schema)
+      val colIdx = model.m.getColIdx(getFeaturesCols().head)
```

Comment:
Is `inputCol` somewhere propagated to `featuresCols`?

Reply:
Good question, it is -> https://github.com/h2oai/sparkling-water/blob/3a86429e56e32d40d51e678ae3e7c0cfae0aecbe/scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OMOJOModel.scala#L251